plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorExecutor.java

```diff
@@ -33,5 +36,13 @@
      * @param editor SQL editor
      * @throws DBException if failed to open the wizard
      */
-    void execute(@NotNull CONTAINER container, @NotNull SQLEditor editor) throws DBException;
+    default void execute(@NotNull CONTAINER container, @NotNull SQLEditor editor) throws DBException {
```

Comment:
Maybe make it `abstract class` then? Also, the name `SQLEditorExecutor` doesn't reflect this interface's purpose at that point.